pkg/cableengine/ipsec/driver.go

```diff
@@ -0,0 +1,22 @@
+package ipsec
+
+import (
+	"github.com/submariner-io/submariner/pkg/types"
+)
+
+// Driver is used by the ipsec engine to actually connect the tunnels
+type Driver interface {
+
+	// Initialize the driver with any state it needs
+	Init() error
+
+	// Returns an array of all the active connections for the given cluster
+	GetActiveConnections(clusterID string) ([]string, error)
+
+	// Connects to the given endpoint and reutrns a string representation of
+	// the IP address of the target endpoint
```

Comment:
How about:

    // ConnectToEndpoint establishes a connection to the given endpoint and returns a string representation of
    // the IP address of the target endpoint.